extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
Hmm, yea, it looks like in production there's an `Age` header which is getting added. It seems to expire after 2 minutes. I was able to recreate the bug in production for 2 minutes getting the following headers until it cleared:

```HTTP
Age:14
Cache-Control:no-cache, must-revalidate

Age:43
Cache-Control:no-cache, must-revalidate

Age:93
Cache-Control:no-cache, must-revalidate

Age:0
Cache-Control:private, s-maxage=0, max-age=0, must-revalidate
(cache cleared here)
```

Those headers differ in dev. They always return
```HTTP
Cache-Control:private, s-maxage=0, max-age=0, must-revalidate
```

Could be worth looking into more. Looks like we may need to ensure `s-maxage=0, max-age=0` gets added? We may just be able to add those [here](https://github.com/Wikia/app/blob/dev/includes/AjaxResponse.php#L130).

What do you think @mklucsarits?